io/EptReader.cpp

```diff
@@ -149,21 +150,44 @@ void EptReader::initialize()
     m_arbiter.reset(new arbiter::Arbiter());
     m_ep.reset(new arbiter::Endpoint(m_arbiter->getEndpoint(m_root)));
     m_pool.reset(new Pool(m_args.threads()));
-    log()->get(LogLevel::Debug) << "Endpoint: " << m_ep->prefixedRoot() <<
-        std::endl;
+    auto& debug(log()->get(LogLevel::Debug));
 
+    debug << "Endpoint: " << m_ep->prefixedRoot() << std::endl;
     m_info.reset(new EptInfo(parse(m_ep->get("ept.json"))));
-    log()->get(LogLevel::Debug) << "Got EPT info" << std::endl;
-
+    debug << "Got EPT info" << std::endl;
+    debug << "SRS: " << m_info->srs() << std::endl;
     setSpatialReference(m_info->srs());
 
     // Figure out our query parameters.
     m_queryBounds = m_args.bounds();
     handleOriginQuery();
 
-    log()->get(LogLevel::Debug) << "Query bounds: " << m_queryBounds <<
-        std::endl;
-    log()->get(LogLevel::Debug) << "Threads: " << m_pool->size() << std::endl;
+    // Figure out our max depth.
+    const double querySpacing(m_args.spacing());
```

Comment:
This is a little confusing in that spacing gets turned into querySpacing and you end up with a new meaning for spacing.

Reply:
Renamed `spacing` to `currentSpacing` in 295c5cede4102b58a3c5b2e865ad3ed2b72ece50 to avoid overloading the term `spacing` with the corresponding stage option.